app/src/main/res/values/strings.xml

```diff
@@ -292,6 +294,9 @@
     <!-- Description for Hinglish (https://en.wikipedia.org/wiki/Hinglish) keyboard subtype with explicit keyboard layout [CHAR LIMIT=25]
          This should be identical to subtype_hi_ZZ aside from the trailing (%s). -->
     <string name="subtype_with_layout_hi_Latn" tools:keep="@string/subtype_with_layout_hi_Latn">Hinglish (<xliff:g id="KEYBOARD_LAYOUT" example="QWERTY">%s</xliff:g>)</string>
+    <!-- Description for Kannada keyboard subtype with explicit keyboard layout [CHAR LIMIT=25]
+         This should be identical to subtype_kn aside from the trailing (%s). -->
+    <string name="subtype_with_layout_knx" tools:keep="@string/subtype_with_layout_knx">Kannada Extended (<xliff:g id="KEYBOARD_LAYOUT" example="QWERTY">%s</xliff:g>)</string>
```

Comment:
Could you add the string in a slightly different way?
My idea would be to use a generic "%s (Extended)" string like `subtype_generic_traditional`. Then if an extended layout is added for another language, there is no need to add yet another string.